File: app/src/main/java/com/project/poupay/view/ContentReportItem.java
package com.project.poupay.view;

import com.project.poupay.R;

public class ContentReportItem {
    private String title;
    private double value;

    public ContentReportItem(String title, double value) {
        this.title = title;
        this.value = value;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public double getValue() {
        return value;
    }

    public void setValue(double value) {
        this.value = value;
    }

    public int getImageID() {
        int id;
        switch (title.toLowerCase()){
            case "ar-condicionado":
            case "arcondicionado":
            case "casaco":
            case "ventilador":
                id = R.drawable.ic_snowflake;
                break;
            case "impressora":
            case "scanner":
            case "multifuncional":
                id = R.drawable.ic_printer;
                break;
            case "passagem":
            case "viagem":
                id = R.drawable.ic_airplane;
                break;
            case "android":
            case "app":
            case "aplicativo":
                id = R.drawable.ic_droid;
                break;
            case "projeto":
            case "arquiteto":
                id = R.drawable.ic_archteture;
                break;

            case "papelaria":
            case "anexo":
            case "escritorio":
                id = R.drawable.ic_clips;
                break;
            case "audio":
            case "musica":
            case "cd":
            case "spotify":
                id = R.drawable.ic_audio;
                break;
            case "livro":
            case "livros":
            case "revista":
            case "hq":
            case "manga":
                id = R.drawable.ic_book;
                break;
            case "advogado":
            case "justiça":
                id = R.drawable.ic_lawer;
                break;
            case "férias":
                id = R.drawable.ic_travel;
                break;
            case "hotel":
            case "motel":
            case "estadia":
                id = R.drawable.ic_hotel;
                break;
            case "eletrodomestico":
            case "eletronico":
            case "utensilho":
                id = R.drawable.ic_eletronic;
                break;
            case "energia":
            case "eletricidade":
                id = R.drawable.ic_power;
                break;
            case "mecânico":
            case "conserto":
            case "encanador":
            case "eletricista":
            case "reparo":
                id = R.drawable.ic_wrench;
                break;
            case "mercado":
            case "supermercado":
            case "feira":
                id = R.drawable.ic_shop;
                break;


            default:
                id = R.drawable.ic_payment_other_36;
                break;

        }


        return id;
    }
}

File: app/src/main/java/com/projeto/poupay/view/ContentReportItem.kt
package com.projeto.poupay.view

import com.projeto.poupay.R


data class ContentReportItem(val title: String, val value: Double) {
    fun getImageID(): Int{
        val id: Int
        when (title.lowercase()) {
            "ar-condicionado", "arcondicionado", "casaco", "ventilador" -> id = R.drawable.ic_snowflake
            "impressora", "scanner", "multifuncional" -> id = R.drawable.ic_printer
            "passagem", "viagem" -> id = R.drawable.ic_airplane
            "android", "app", "aplicativo" -> id = R.drawable.ic_droid
            "projeto", "arquiteto" -> id = R.drawable.ic_archteture
            "papelaria", "anexo", "escritorio" -> id = R.drawable.ic_clips
            "audio", "musica", "cd", "spotify" -> id = R.drawable.ic_audio
            "livro", "livros", "revista", "hq", "manga" -> id = R.drawable.ic_book
            "advogado", "justiça" -> id = R.drawable.ic_lawer
            "férias" -> id = R.drawable.ic_travel
            "hotel", "motel", "estadia" -> id = R.drawable.ic_hotel
            "eletrodomestico", "eletronico", "utensilho" -> id = R.drawable.ic_eletronic
            "energia", "eletricidade" -> id = R.drawable.ic_power
            "mecânico", "conserto", "encanador", "eletricista", "reparo" -> id = R.drawable.ic_wrench
            "mercado", "supermercado", "feira" -> id = R.drawable.ic_shop
            "pix" -> id = R.drawable.ic_pix_18
            "dinheiro" -> id = R.drawable.ic_money_18
            "cartão de crédito" -> id = R.drawable.ic_card_18
            "outro" -> id = R.drawable.ic_baseline_plans_local_atm_24
            else -> id = R.drawable.ic_payment_other_36
        }
        return id
    }
}